Sources/NIOExtras/LengthFieldBitLength.swift

```diff
@@ -0,0 +1,69 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2021 Apple Inc. and the SwiftNIO project authors
```

Comment:
sorry for being annoying here but given that this is a new file, it should be just `(c) 2021`, unless I missed something and you actually took some code from elsewhere that had `2017 - ...`